ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-cpou-upgrade-from-4.12.yaml

```diff
@@ -84,18 +104,15 @@ tests:
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-azure-ipi-workers-rhel8
-- as: baremetalds-ipi-ovn-ipv4-fips-f28
-  cron: 22 2 5 * *
+- as: gcp-ipi-basecap-none-additionalcaps-f14
```

Comment:
I see there are two profiles on gcp, i think only profile on each platform should be enough.

Reply:
it's really difficult for me to find jobs which can cover all key features and not have duplicate platform, here are the available profiles in openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:
```
[Running] python -u "/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py"
- as: aws-c2s-ipi-disc-priv-fips-f28
- as: aws-ipi-disc-priv-mtu-localzone-fips-f28
- as: aws-ipi-disc-priv-sno-basecap-none-f28
- as: aws-ipi-disc-priv-sts-efs-f28
- as: aws-ipi-disc-priv-sts-ep-fips-f28
- as: aws-ipi-network-mtu-localzone-f28
- as: aws-ipi-network-mtu-localzone-sdn-f28
- as: aws-ipi-private-shared-vpc-phz-sts-f28
- as: aws-ipi-shared-vpc-phz-f28
- as: aws-ipi-shared-vpc-phz-sts-fips-f28
- as: aws-sc2s-ipi-disc-priv-fips-f28
- as: aws-usgov-ipi-disc-priv-f28
- as: aws-usgov-ipi-private-f28
- as: aws-usgov-ipi-private-sts-f28
- as: azure-ipi-disc-f28
- as: azure-ipi-disc-fullyprivate-f28
- as: azure-ipi-vmgenv1-f28
- as: azure-mag-ipi-fips-f28
- as: azure-mag-ipi-fullyprivate-f28
- as: azure-stack-ipi-proxy-fips-f28
- as: azure-stack-upi-f28
- as: baremetalds-ipi-ovn-ipv4-fips-f28
- as: baremetalds-ipi-ovn-ipv6-fips-f28
- as: baremetalds-ipi-ovn-dualstack-fips-f28
- as: gcp-ipi-basecap-none-additionalcaps-f28
- as: gcp-ipi-disc-priv-oidc-fips-f28
- as: gcp-ipi-disk-encryption-f28
- as: gcp-ipi-mini-perm-custom-type-f28
- as: gcp-ipi-ovn-ipsec-f28
- as: gcp-ipi-proxy-etcd-encryption-f28
- as: gcp-ipi-proxy-private-f28
- as: gcp-ipi-sdn-f28
- as: gcp-ipi-xpn-techpreview-fips-f28
- as: gcp-upi-f28
- as: gcp-upi-private-xpn-ingress-glb-f28
- as: gcp-upi-private-xpn-sdn-f28
- as: gcp-upi-xpn-f28
- as: ibmcloud-ipi-dedicatedhost-static-f28
- as: ibmcloud-ipi-mini-perm-f28
- as: ibmcloud-ipi-private-fips-f28
- as: ibmcloud-ipi-rt-f28
- as: nutanix-ipi-disc-rt-f28
- as: nutanix-ipi-proxy-fips-f28
- as: vsphere-ipi-disc-f28
- as: vsphere-ipi-disk-encryption-tang-fips-f28
- as: vsphere-ipi-external-lb-post-f28
- as: vsphere-ipi-ovn-etcd-encryption-f28
- as: vsphere-ipi-ovn-ipsec-f28
- as: vsphere-ipi-proxy-fips-f28
- as: vsphere-ipi-sdn-f28
- as: vsphere-upi-disc-secureboot-fips-f28
- as: vsphere-upi-f28
```

Reply:
only `aws-ipi-disc-priv-sno-basecap-none-f28` and `gcp-ipi-basecap-none-additionalcaps-f28` can cover `basecap`, but `aws-ipi-disc-priv-sno-basecap-none-f28`  is a disc cluster which not good for multi-hop job.

Reply:
`gcp-upi-private-xpn-sdn-f14` covers `upi` and `sdn` which can also reduce profile number. if we don't use this profile, we have to choose 2 other profile to cover `upi` and `sdn` because there are no other single profiles can cover them at same time.

Reply:
if this profile `gcp-ipi-basecap-none-additionalcaps-f14` is only for covering none-caps, i think we could drop it in eus upgrade because the feature has been covered in chain upgrade(for multi-hops) and to-be covered in canary upgrade(for paused worker pool). cc @shellyyang1989 to confirm

Reply:
BTW, I want to submit a new pr after https://github.com/openshift/release/pull/55328 and this pr get merged, the new pr will rename eus related steps, chain and file names to cpou, for example, rename `- chain: openshift-upgrade-qe-test-eus` to `- chain: openshift-upgrade-qe-test-cpou`

Reply:
@shellyyang1989 Please help confirm if we can remove `gcp-ipi-basecap-none-additionalcaps-f14` from 4.12 -> 4.14 eus upgrade?

Reply:
Thanks for asking. Let's loop @jianlinliu @gpei to discuss if we need to cover `capability` in EUS upgrade?

Reply:
IMHO, EUS upgrade seems to be a relatively customer preferred upgrade approach for self-hosted OCP. I'm not sure if there are differences between chain upgrade, canary upgrade and eus upgrade in terms of capability. I personally think we can keep it.

Reply:
From my understanding, `capability` is only a cluster configurations, it does not care too much about eus upgrade or non-eus, once any capacity is enabled in source cluster, the capacity would be always enabled in all the upgrade path. So both (cover it or not cover it) is okay for me. But I am not a fan to cover many profiles in eus upgrade, "one profile on each platform" is preferred for me.

Reply:
>  I'm not sure if there are differences between chain upgrade, canary upgrade and eus upgrade in terms of capability. I personally think we can keep it.

hmm, my original concern is that `do we really need two profiles on gcp`? if you'd like to cover capability in eus, i think we could enable it on other profiles easily by setting `ADDITIONAL_ENABLED_CAPABILITIES` and `BASELINE_CAPABILITY_SET`, no need an extra dedicated profile for `none-caps`, right?

Reply:
>if we need to cover capability in EUS upgrade

I'm not quite familiar with our current upgrade profile configurations, but for me the question seems just to be the strategy we choose to run in EUS upgrade, if we need/want to cover all the key configurations in EUS upgrade, then maybe we need the similar testing coverage for EUS upgrade profiles, of course including `capability`, if not, then we can just choose the proper and minimal coverage unless there's existing bugs about `capability` setting broken during the EUS upgrade, just some personal suggestions.

Reply:
> my original concern is that do we really need two profiles on gcp? if you'd like to cover capability in eus, i think we could enable it on other profiles easily by setting ADDITIONAL_ENABLED_CAPABILITIES and BASELINE_CAPABILITY_SET, no need an extra dedicated profile for none-caps, right?

ah, I misunderstood. Yeah, ideally one profile on each platform is perfect if all key features can be covered. For this case, i'm personally fine with enabling cap on other profiles. It will slightly obey our rule that using e2e profiles but it will decrease the number of jobs and save cost relatively.

Reply:
removed `gcp-ipi-basecap-none-additionalcaps` and enabled `ADDITIONAL_ENABLED_CAPABILITIES` and `BASELINE_CAPABILITY_SET` ,in `gcp-upi-private-xpn-sdn-f14`

Reply:
removed basecap job from 4.12 -> 4.14 eus upgrade, more comments can be found in https://github.com/openshift/release/pull/55502#discussion_r1735552302